akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/scaladsl/ClusterShardingPersistenceSpec.scala

```diff
@@ -37,7 +37,7 @@ object ClusterShardingPersistenceSpec {
   val config = ConfigFactory.parseString(
     """
       akka.loglevel = DEBUG
-      #akka.persistence.typed.log-stashing = on
+      akka.persistence.typed.log-stashing = on
```

Comment:
Why was it commented and now uncommented?

Reply:
leftover from debugging, changed back in 9596cd6